lighthouse-core/audits/first-meaningful-paint.js

```diff
@@ -242,7 +242,10 @@ class FirstMeaningfulPaint extends Audit {
       }
     });
 
-    const paintAfterMSLayout = evts.paints.find(e => e.ts > mostSignificantLayout.ts);
+    let paintAfterMSLayout;
```

Comment:
Then the first-meangingful-paint will be defined by the firstContentfulPaint alone, if it exists.